### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by MaksymMaliuk**
[DEMO LINK](https://MaksymMaliuk.github.io/react_people-table-basics/)

### CODE REVIEW

File: src/Pages/NotFoundPage.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const NotFoundPage: React.FC = () => {
+  return (
```
Comment (Stepan-TS) [inline src/Pages/NotFoundPage.tsx:4]: unnecessary return)

File: src/Pages/HomePage.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const HomePage: React.FC = () => {
+  return (
```
Comment (Stepan-TS) [inline src/Pages/HomePage.tsx:4]: unnecessary return)

File: src/Pages/PeoplePage.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+      <div className="block">
+        <div className="box table-container">
+          {isError && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              Something went wrong
+            </p>
+          )}
+
+          {(!peopleTable.length && !isLoading) && (
```
Comment (Stepan-TS) [inline src/Pages/PeoplePage.tsx:59]: it might be better to put this expression into a variable for better readability)

**Review by Stepan-TS — APPROVED** [review]
good job, but i suggested fix some code style)

File: src/Pages/HomePage.tsx (hunk 2 above)
Comment (OleksiiBlezniuk) [inline src/Pages/HomePage.tsx:4]:
```suggestion
export const HomePage: React.FC = () => (
```

File: src/Pages/PeoplePage.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Person } from '../types/Person';
+import { PeopleTable } from '../components/PeopleTable';
+import { Loader } from '../components/Loader';
+
+const findParent = (people: Person[], name: string) => {
+  return people.find(person => person.name === name);
```
Comment (OleksiiBlezniuk) [inline src/Pages/PeoplePage.tsx:8]: I guess will be more logical to set null if nothing is found, but it's not critical

I mean - undefined tell us that somehow data wasn't set here due to mistake, error or something like that
null - tell us that there shouldn't be anything intentionally

**Review by OleksiiBlezniuk — CHANGES_REQUESTED** [review]
Let's render table only when there is no errors
Also let's show only one error message at one time, if there was error while loading - show error about it, not about array is empty
Screenshot 2023-04-19 at 12 14 54

**Review by OleksiiBlezniuk — APPROVED** [review]
GJ!

**Pull request closed without merging**